src/documents/IoCTestingFramework.md

```diff
@@ -388,6 +665,7 @@ fluid.defaults("fluid.tests.asyncTester", {
                 func: "fluid.tests.changeField",
                 args: ["{asyncTest}.dom.textField", "{asyncTester}.options.newTextValue"]
             }, {
+                 // old-fashioned "listenerMaker" - discouraged in modern code
```

Comment:
Should the example be changed to not include the old-fashioned "listenerMaker" instead?